### PR TITLE
Fixes PL-1 and PL-3 Restricting team creation 

### PR DESCRIPTION
**Pull request by coreyhulen**

### CODE REVIEW

**Comment by jwilander** [comment]
+1

File: docker/0.6/config_docker.json [hunk 1]
```diff
@@ -94,6 +94,8 @@
         "HelpLink": "/static/help/configure_links.html",
         "ReportProblemLink": "/static/help/configure_links.html",
         "TourLink": "/static/help/configure_links.html",
-        "DefaultThemeColor": "#2389D7"
+        "DefaultThemeColor": "#2389D7",
+        "DisableTeamCreation": true,
```
Comment (crspeller) [inline docker/0.6/config_docker.json]: Why are we disabling team creation in the docker image?

Comment (crspeller) [inline docker/0.6/config_docker.json]: Also this is the 0.6 configuration file. It does not need changes.

**Comment by crspeller** [comment]
Needs change to configuration file for docker dev.

**Comment by coreyhulen** [comment]
@crspeller please re-review.  I've made the changes suggested.

**Comment by crspeller** [comment]
Looks good!

**Pull request merged**